src/components/line-chart-audification/line-chart-audification.component.html

```diff
@@ -1,3 +0,0 @@
-<div role="button" (focus)="playMelody()" (blur)="disposeMelody && disposeMelody()">
```

Comment:
There will be a template/style to this component. (In #14: [line-chart-audification.component.html](https://github.com/googleinterns/guide-doge/blob/16a6f9c17c1624c219001a0f7bb425e95f5a89ea/src/components/line-chart-audification/line-chart-audification.component.html)) Though, I agree that it would semantically look better with a directive if a template was not needed.